experiments/customui/parent.js

```diff
@@ -319,7 +319,209 @@ var ex_customui = class extends ExtensionCommon.ExtensionAPI {
       setWebextFrameDynamicVisibility(frame, options);
       setWebextFrameDynamicDimension(frame, options, "height", 100);
     }
-    
+
+    // get the grid-template css rule of a container
+    const getGridTemplate = function(container,document) {
```

Comment:
This does not take the element's style into account, correct?

If so, multiple add-ons doing this would cause issues? Maybe even multiple sidebars (I did not do any testing)?

Reply:
I don't thinks I fully understand your remark. If I understand your question, then this does take the element's style into account. That is essentially what it does.
Also I don't see why this would cause an issue. Maybe the timing of multiple added sidebars, but as they are vertically stacked at the moment, this is not an issue either.

Reply:
It checks style sheets, but not a `style` attribute on the element itself.